### PR TITLE
Suggests to depends

### LINKED ISSUE #135: Make suggests depends
We need packages like `cmdstanr` to be installed - moving from Suggest to Depends would fix that.

### PR DESCRIPTION
**Pull request by seabbs**
Closes #135 and updates the author list. Removed the `cph` as I don't think its accurate?

### CODE REVIEW

**Comment by sbfnk** [comment]
> Removed the cph as I don't think its accurate?

It isn't - tanks for spotting.

That said, maintaining this could become annoying (what is the relationship of the R package to the course content? has everybody who has contributed to the course contributed to the package? etc). Perhaps we should just have a generic "NFIDD course contributors" author or the like? Counterpoint being that it might be useful to link the ORCID?

**Comment by sbfnk** [comment]
Given that we're only using this for dependency management and data loading it really is an abuse of R package infrastructure and probably could be solved more elegantly at a later stage.

**Comment by seabbs** [comment]
> "NFIDD course contributors" author or the like? Counterpoint being that it might be useful to link the ORCID?

Yeah I think this is the right call

**Review by sbfnk — APPROVED** [review]

File: DESCRIPTION [hunk 1]
```diff
@@ -4,11 +4,10 @@ Title: Material to support course on nowcasting and forecasting of infectious
...
     person(
-      given = "Sebastian",
-      family = "Funk",
-      role = c("cre", "cph", "aut"),
-      email = "sebastian.funk@lshtm.ac.uk",
-      comment = c(ORCID = "0000-0002-2842-3406")
+      given = "NFIDD course contributors",
+      family = "",
+      role = c("cre", "aut"),
+      email = "epiforecasts@gmail.com"
```
Comment (sbfnk) [inline DESCRIPTION:10]: do we need an email? I don't think anyone monitors that.

Comment (sbfnk) [reply DESCRIPTION:10]: just checked - 5 emails since 2022. Phew, nothing missed.

Comment (seabbs) [reply DESCRIPTION:10]: I wasn't clear you could avoid having an email?

**Pull request merged**